skylarkstruct/struct.go

```diff
@@ -223,7 +217,11 @@ func (s *Struct) Attr(name string) (skylark.Value, error) {
 		return s.entries[i].value, nil
 	}
 
-	// TODO(adonovan): to_{json,proto} are deprecated (b/36412967).
+	// TODO(adonovan): there may be a nice feature for core
+	// skylark.Value here, especially for JSON, but the current
+	// features are incomplete and underspecified.
+	//
+	// to_{json,proto} are deprecated, appropriately; see Google issue b/36412967.
```

Comment:
So 1. we outside of Google can not see any of your internal issues. Referring to them is not the nicest thing to do.
2. This is the first reference I found for `to_`* deprecation. While I can think of a few reasons why - it is unclear what the actual reasons are. Could you please create a public issue for that topic somewhere.
3. Do you already have a Skylark (still waiting on the rename to happen) implementation for `json.dumps`?

Summoning @dslomov